subworkflows/local/input_check.nf

```diff
@@ -91,8 +91,7 @@ def reset_combined_map(LinkedHashMap meta, sun.nio.fs.UnixPath f_reads, sun.nio.
 
 }
 
-def check_file_exists(String file_path){
-
+def check_file_exists(def file_path){
```

Comment:
Adding `"format": "file-path"` to the `schema_input.json` means that files come in not as strings but as instances of Nextflow file objects, which meant this function was unable to be called if restricted to only strings.

Reply:
awesome! Thank you!